bin/convert_hf_to_gguf.py

```diff
@@ -22,7 +22,7 @@
     sys.path.insert(1, str(Path(__file__).parent / 'gguf-py'))
 import gguf
 
-from convert import HfVocab
+from bin.convert import HfVocab
```

Comment:
1. We'll need to check with `@ggerganov` (not pinging yet) about moving stuff around, I don't know what assumptions people might be making about paths...
2. Generally, using top-level names like `bin` or `convert` is rather unhygienic, it's asking for conflicts... With your move a relative import should work, but first...


...what was the motivation for the move?

Reply:
Ah, having a subdir makes it a tad easier to get the package working with poetry, but certainly not a requirement. I'll revert the locations.

Reply:
However, we'll still need to convert the filenames from kebab-case to snake_case, otherwise they can't be treated and run as modules.